react-native-libraries.json

```
@@ -10230,5 +10230,11 @@
     "android": true,
     "web": true,
     "expo": true
+  },
+  {
+    "githubUrl": "https://github.com/shyambhongle/react-native-video-duration",
+    "examples": ["https://github.com/shyambhongle/react-native-video-duration/example"],
+    "ios": true,
+    "android": true,
```

Hey @shyambhongle, thank you for adding new entry to the directory! 👍 

```suggestion
    "android": true
```

The trailing comma at the end of entry is causing CI to fail, with that fixed, changes should be ready to ship. 🙂